src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-navDropdown/Components/index.test.tsx

```diff
@@ -21,26 +21,45 @@ describe('<DashboardNavigationDropdown />', () => {
     wrapper = mount(<DashboardNavigationDropdown {...props} />);
   });
 
-  it('allows us to set Name props', () => {
-    expect(wrapper.props().Name).toEqual('name');
-    wrapper.setProps({ Name: 'different name' });
-    expect(wrapper.props().Name).toEqual('different name');
+  afterEach(() => {
+    jest.clearAllMocks();
   });
 
-  it('allows us to set Options props', () => {
-    expect(wrapper.props().Options).toEqual([{ title: 'title' }]);
-    wrapper.setProps({
-      Options: [{ title: 'second title' }, { title: 'third title' }],
-    });
-    expect(wrapper.props().Options).toEqual([
-      { title: 'second title' },
-      { title: 'third title' },
-    ]);
+  it('it should render Name prop inside NavigationHeading surrounded by static icons', () => {
+    expect(
+      wrapper
+        .find(NavigationHeading)
+        .children()
+        .find(Icon).length,
+    ).toBe(2);
+    expect(wrapper.find(AppIcon.StudentIcon).length).toBe(1);
+    expect(wrapper.find(AppIcon.FilledArrowDownIcon).length).toBe(1);
+    expect(wrapper.props().Name).toBe(props.Name); // ? unsure on how to write this part
+    // expect(wrapper.find(NavigationHeading).children().text()).toBe(props.Name); // ? initially wrote it like this but can't figure out how to trim away the icon text name
```

Comment:
```suggestion
    expect(
      wrapper
        .find(NavigationHeading)
        .children()
        .text(),
    ).toBe(`Student Icon${props.Name}FilledArrowDownIcon`);
```

A little workaround if you don't want to trim out the Icon names.